unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/TheGraph/TheGraph.cs

```diff
@@ -90,6 +92,32 @@ public Promise<List<Land>> QueryLands(string tld, string address, float cacheMax
         return promise;
     }
 
+    public Promise<double> QueryMana(string address)
```

Comment:
Could we add something to explicitly point out that this is for matic/polygon chain?
Not sure what would be the best way to go, changing the method name or adding a new argument to the method like `bool usePolygonChain` and use one url or the other depending on that.

Reply:
I'm gonna rename it to `QueryPolygonMana` and once we port the retrieval of `MainNet/Ropsten` to Unity we can create another query (similar to how Lands are requested)